
# HG changeset patch
# User Mike Hommey <mh+mozilla@glandium.org>
# Date 1692737029 0
# Node ID 45044573c4398ac2a2a3a8b798a5ce413342b0aa
# Parent  faca0b32a21483f72d985fc2a7963e0178f3bbfd
Bug 1849718 - Remove redundant `use`. r=canaltinova

MarkerOptions is already imported via `pub use options::*;`.

Differential Revision: https://phabricator.services.mozilla.com/D186664

diff --git a/tools/profiler/rust-api/src/marker/mod.rs b/tools/profiler/rust-api/src/marker/mod.rs
--- a/tools/profiler/rust-api/src/marker/mod.rs
+++ b/tools/profiler/rust-api/src/marker/mod.rs
@@ -122,17 +122,16 @@ pub mod options;
 pub mod schema;
 
 pub use options::*;
 pub use schema::MarkerSchema;
 
 use crate::gecko_bindings::{bindings, profiling_categories::ProfilingCategoryPair};
 use crate::json_writer::JSONWriter;
 use crate::marker::deserializer_tags_state::get_or_insert_deserializer_tag;
-use crate::marker::options::MarkerOptions;
 use serde::{de::DeserializeOwned, Deserialize, Serialize};
 use std::os::raw::c_char;
 
 /// Marker API to add a new simple marker without any payload.
 /// Please see the module documentation on how to add a marker with this API.
 pub fn add_untyped_marker(name: &str, category: ProfilingCategoryPair, mut options: MarkerOptions) {
     if !crate::profiler_state::can_accept_markers() {
         // Nothing to do.

